definitions-terms.md

```diff
@@ -0,0 +1,108 @@
+# Definitions and Terms
+
+A collection of definitions and terms used within this repository.
+
+## TOC
+
+* [Fingerprint](#fingerprint)
+* [Fully Qualified Artifact Reference](#registry-fully-qualified-reference)
+* [Multi-Tenant Registry](#multi-tenant-registry)
+* [OCI Annotations](#oci-annotations)
+* [OCI Descriptor](#oci-descriptor)
+* [OCI Index](#oci-index)
+* [OCI Manifest](#oci-manifest)
+* [Registry](#registry)
+* [SBoM](#sbom)
+* [Subject](#subject)
+* [Tag](#tag)
+
+## Artifact / Image
+
+Artifacts are a generalization of how images are stored in an instance of the [OCI Distribution Spec][oci-distribution-spec]. An [OCI Image][oci-image] is a type of [OCI Artifact][oci-artifact]. When referencing artifacts, within a [fully qualified reference](#registry-fully-qualified-reference), the artifact is right most element of the namespace, combined with the `:tag`.  
+Reference: [OCI Artifacts][oci-artifact]  
+![](./media/artifact-ref.png)  
+
+
+## Fingerprint
+
+A short identifier of a given public key.  
+Reference: [Fingerprint][fingerprint]
+
+## Multi-Tenant Registry
+
+A collection of unique [registries](#registry), stored within an instance of an [OCI Distribution Spec][oci-distribution-spec]. Registry operators use one of two methods to uniquely identify an org. In either case, information within that org is considered unique to the org, including all key references and authentication.
+
+## OCI Annotations
+
+A key-value map that can be associated with [OCI Descriptors][oci-descriptor] and [OCI Image manifests][oci-manifest].  
+OCI spec reference: [OCI Annotations][oci-annotations]
+
+## OCI Descriptor
+
+A structure describing content, including the media type, a content-addressable digest, the size, and other properties. Descriptors are used to describe layers and configuration in a manifest.  
+OCI spec reference: [OCI Descriptor][oci-descriptor]
+
+## OCI Index
+
+A higher-level collection of [image manifests][oci-manifest], or other [oci indexes][oci-index], typically used to describe platform-specific (architecture and operating-system) images that can be identified collectively and referred to together. The specific image manifests are identified by modified [descriptors][oci-descriptor] with additional properties and restrictions. [OCI Artifacts][oci-artifact] takes this a step further and provides for an OCI Index to reference a collection of artifacts, such as an image, a deployment chart, its SBoM and deployment configuration references.  
+OCI spec reference: [OCI Image Index][oci-index]
+
+## OCI Manifest
+
+A description of an [artifact][oci-artifact] within an instance of an [OCI Distribution][oci-distribution-spec] complaint registry. The manifest references optional configuration and blobs (layers) as content-addressable references ([descriptors][oci-descriptor]).  Artifacts may be container images, helm charts or other artifact types that may be signed and stored in a registry.    
```

Comment:
The manifest is defined by the [image spec](https://github.com/opencontainers/image-spec/blob/master/manifest.md), not by the distribution spec.  We should remove the language about "within an isntance of an OCI Distribution compliant registry", since a manifest is valid both inside and outside a registry.

Reply:
The image spec and distribution spec have things intertwined between them. We have a work-item to detangle these, likely refactoring the schemas as a top level object, that artifacts use to represent objects that get persisted in a distribution. The image-spec is one of many artifact types that use these schemas. 
- [Need a Working Group for Updating the OCI distribution-spec](https://github.com/opencontainers/artifacts/issues/9)
- [Need a working group for updating the image spec](https://github.com/opencontainers/artifacts/issues/10)

Reply:
I still think we should remove the language about "within an instance of an OCI Distribution compliant registry".  The manifest is valid both inside and outside the registry; the location of the manifest does not play into its validity.